addons/im_livechat/static/src/core/web/suggestion_service_patch.js

```diff
@@ -11,8 +11,15 @@ patch(SuggestionService.prototype, {
             ? [...super.getSupportedDelimiters(...arguments), [":"]]
             : super.getSupportedDelimiters(...arguments);
     },
+    async fetchSuggestions({ delimiter, term }, { thread } = {}) {
+        if (thread.type === "livechat" && delimiter === "#") {
+            return;
+        }
+        return super.fetchSuggestions(...arguments);
+    },
```

Comment:
Could be nice a test for that, so that follow-up PR to clean code can ensure the behaviour is kept.
Since this test would assert a negative, it can be done with asserting it works in another non-livechat channel. Doing the same action in different channels and have different behaviours (one has the feature, the other not) is a relatively good test.